.travis.yml

```diff
@@ -0,0 +1,20 @@
+dist: bionic
+
+install: 
+  - bin/system-setup
+
+jobs:
+  fast_finish: true # fail once any job fail
+
+env:
+  - ARCHES=aarch64
+  - ARCHES=arm
+  - ARCHES=i386
+  - ARCHES=x86_64
+  - ARCHES=powerpc
+  - ARCHES=ppc64
+  - ARCHES=ppc64le
+
+script:
+  - QUIET=1 sudo -E bash bin/build-release daily
```

Comment:
fwiw, `sudo QUIET=1 ./bin/build-release daily`.
I guess maybe you were using '-E' in order to pass other things through the environment.

and i'd drop the bash.  and trust the '#!' to do the right thing.